bpmsuite/bpmsuite70-businesscentral-monitoring-ephemeral.json

```diff
@@ -0,0 +1,506 @@
+            "name": "KIE_ADMIN_PWD",
+            "from": "[a-zA-Z]{6}[0-9]{1}!",
+            "generate": "expression",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Controller User",
+            "description": "KIE server controller username (adds the user)",
+            "name": "KIE_SERVER_CONTROLLER_USER",
+            "value": "controllerUser",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Controller Password",
+            "description": "KIE server controller password (adds the user)",
+            "name": "KIE_SERVER_CONTROLLER_PWD",
+            "value": "controllerPassword",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server User",
+            "description": "KIE execution server username (Sets the org.kie.server.user system property, used by the embedded controller)",
+            "name": "KIE_SERVER_USER",
+            "value": "executionUser",
+            "required": false
+        },
+        {
+            "displayName": "KIE Server Password",
+            "description": "KIE execution server password (Sets the org.kie.server.pwd system property, used by the embedded controller)",
+            "name": "KIE_SERVER_PWD",
+            "value": "executionPassword",
+            "required": false
+        },
+        {
+            "displayName": "Smart Router listening port",
+            "description": "Port in which the smart router server listens (router property org.kie.server.router.port)",
+            "name": "KIE_ROUTER_PORT",
+            "value": "9000",
+            "required": false
+        },
+        {
+            "displayName": "Smart Router external URL",
+            "description": "Public URL where the router can be found. Format http://<host>:<port> (router property org.kie.server.router.url.external)",
+            "name": "KIE_ROUTER_URL_EXTERNAL",
+            "example": "http://my-app-smartrouter-projectName.osurl:9000",
+            "required": false
+        },
+        {
+            "displayName": "Smart Router ID",
+            "description": "Router ID used when connecting to the controller (router property org.kie.server.router.id)",
+            "name": "KIE_ROUTER_ID",
+            "value": "kie-server-router",
+            "required": false
+        },
+        {
+            "displayName": "Smart Router name",
+            "description": "Router name used when connecting to the controller (router property org.kie.server.router.name)",
+            "name": "KIE_ROUTER_NAME",
+            "value": "KIE Server Router",
+            "required": false
+        },
+        {
+            "displayName": "Smart Router controller URL",
+            "description": "URL to access the controller (standalone or embedded in the business central) (router property org.kie.server.controller)",
+            "name": "KIE_ROUTER_CONTROLLER_URL",
```

Comment:
Here it would be great if it could be possible to automatically set this variable to the external route of the controller embedded in the BC. It would be still possible to change it to point to the standalone controller

Reply:
The router image should change to lookup the service (instead of depending on an external url), as we do in other places (for example, looking up a maven repo service). I will add it to the router image scripts and update this template after the fact. I won't hold up this PR with it, though.

Reply:
Thanks, @errantepiphany